src/atomic.py

```diff
@@ -79,6 +79,10 @@ def look_at_point(x_node, y_node, z_node):
 
 def look_at_face_point(x, y, z):
 	evl.look_at_point(x, y, z)
+	#evl.gaze_at_point(x, y, z)
```

Comment:
I don't understand what you are doing here. This would seem to break the voice commands, where you tell the robot to   turn and look to the left or right or up or down.   Did you make some other changes to support this ability, or is the chatbot now broken?

Please have other people review your code before committing changes.